.travis.yml

```diff
@@ -1,5 +1,7 @@
 # https://travis-ci.org
 
+sudo: required
```

Comment:
mm, well its right don't is a issdue of the main proyects, but as we saw with all my failed commits in the irc affects the forks, anyway i only can opine about this, then remove or not?